daprdocs/layouts/shortcodes/dapr-version.html

```diff
@@ -0,0 +1 @@
+1.4.0
```

Comment:
should this file (and consequently its shortcodes) be named "dapr-latest-released-version" to be more explicit?

Reply:
I can do that.